src/tests/lib/repositories_v2/generic_sqlalchemy_mock_builder.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABCMeta, abstractmethod
+
+
+class GenericQueryMockBuilder(metaclass=ABCMeta):
```

Comment:
This is not necessary since there is no different implementation in a near future.

Reply:
done.